README.md

```diff
@@ -198,3 +198,84 @@ func Function(
 	
 }
 ```
+
+### Sync Tool and Operator Types
+
+To switch to using just the runtime client and to remove dependencies on individual operators, the sync tool will periodically copy over types from operator repos to this repo. It works by cloning the operator repo, applying changes in the config file, then copying it into this repo.
+
+#### Running
+
+To run the sync tool, use the makefile target.
+
+```
+make lib-sync
+```
+
+If the sync fails while adding a new set of operator types, remove the synced directory from `schemes/` and rerun the sync.
+
+#### Configuration
+
+Config files for the sync tool live in the [internal/sync/configs](./internal/sync/configs/) directory. A good example of all the features available is in the [nvidia-config.yaml](./internal/sync/configs/nvidia-config.yaml) file.
+
+Each config is a YAML document with a top level list of objects that follow this form:
+
+```yaml
+- name: operator-repo # 1
+  sync: true # 2
+  repo_link: "https://github.com/operator/repo" # 3
+  branch: main # 4
+  remote_api_directory: pkg/apis/v1 # 5
```

Comment:
looking at the examples below is this (and the next line's) configs needed? They seem to be included in the url anyway

Reply:
@achuzhoy Yes the `remote_api_directory` and `local_api_directory` are necessary. The url is just the url for git clone